foxmlToSolr.xslt

```diff
@@ -225,6 +226,15 @@
               <xsl:with-param name="content" select="document(concat($PROT, '://', encoder:encode($FEDORAUSER), ':', encoder:encode($FEDORAPASS), '@', $HOST, ':', $PORT, '/fedora/objects/', $PID, '/datastreams/', @ID, '/content'))"/>
             </xsl:apply-templates>
           </xsl:when>
+          <!-- JSON to document objects -->
+          <xsl:when test="@CONTROL_GROUP='M' and foxml:datastreamVersion[last() and @MIMETYPE='application/json']">
+            <xsl:variable name="json">
+              <xsl:value-of select="java:ca.discoverygarden.gsearch_extensions.FedoraUtils.getRawDatastreamDissemination($PID, @ID, concat($PROT, '://', $HOST, ':', $PORT, '/fedora'), $FEDORAUSER, $FEDORAPASS)"/>
```

Comment:
Also... could use the other syntax, if we wanted to inline...:
```xsl
<xsl:when test="[... the current test ...]"
  xmlns:dgi-e="xalan://ca.discoverygarden.gsearch_extensions">
  <xsl:variable name="json" select="dgi-e:FedoraUtils.getRawDatastreamDissemination($PID, @ID, concat($PROT, '://', $HOST, ':', $PORT, '/fedora'), $FEDORAUSER, $FEDORAPASS)"/>
  <xsl:apply-templates select="foxml:datastreamVersion[last()]">
    <xsl:with-param name="content" select="dgi-e:JSONToXML.convertJSONToXML($json)"/>
  </xsl:apply-templates>
</xsl:when>
```
or even:
```xsl
<xsl:when test="[... the current test ...]"
  xmlns:dgi-e="xalan://ca.discoverygarden.gsearch_extensions">
  <xsl:apply-templates select="foxml:datastreamVersion[last()]">
    <xsl:with-param name="content" select="dgi-e:JSONToXML.convertJSONToXML(dgi-e:FedoraUtils.getRawDatastreamDissemination($PID, @ID, concat($PROT, '://', $HOST, ':', $PORT, '/fedora'), $FEDORAUSER, $FEDORAPASS))"/>
  </xsl:apply-templates>
</xsl:when>
```
... shrug?